bin/codeclimate-brakeman

```diff
@@ -13,7 +13,11 @@ end
 command = "/usr/src/app/bin/brakeman --quiet --format codeclimate"
 
 if engine_config["include_paths"]
-  command += " --only-files " + engine_config["include_paths"].compact.join(",").shellescape
+  command << " --only-files #{engine_config["include_paths"].compact.join(",").shellescape}"
+end
+
+if system("test -r /dev/stdout")
```

Comment:
Both would probably be true or false together, but is `-w` more intention-revealing? You do intend to write to it right?